mods/d2k/rules/vehicles.yaml

```diff
@@ -328,8 +329,26 @@ devastator:
 	Explodes:
 		Weapon: UnitExplodeLarge
 		EmptyWeapon: UnitExplodeLarge
+		RequiresCondition: !overload
 	SpawnActorOnDeath:
 		Actor: devastator.husk
+	Explodes@OVERLOAD:
+		Weapon: PlasmaExplosion
+		EmptyWeapon: PlasmaExplosion
+		RequiresCondition: meltdown
+	GrantConditionOnDeploy@REACTOR:
+		DeployedCondition: overload
+		CanUndeploy: false
+	WithIdleOverlay@OVERLOAD
```

Comment:
Missing `:` at the end.

Reply:
Fixed. Weird that it did work without it.